### ISSUE TITLE
openssh choco installation fails.

### ISSUE THREAD

**Issue by cowlinator** [DarwinJS/ChocoPackages#43]
Installation failed. 

choco install -y openssh

resulted in an error.  
The log at "C:\ProgramData\chocolatey\logs\chocolatey.log" contains:

2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: Everything is Ok
2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: 
2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: Folders: 1
2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: Files: 18
2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: Size: 6825118
2017-08-15 19:02:13,190 7720 [INFO ] - VERBOSE: Compressed: 3135596
2017-08-15 19:02:13,221 7720 [DEBUG] - $exitCode was passed null
2017-08-15 19:02:13,221 7720 [DEBUG] - Command ['C:\ProgramData\chocolatey\tools\7z.exe' x -aoa -bd -bb1 -o"C:\Users\buildbot\AppData\Local\Temp\chocolatey\OpenSSHTemp" -y "C:\ProgramData\chocolatey\lib\openssh\tools\OpenSSH-Win64.zip"] exited with '0'.
2017-08-15 19:02:13,221 7720 [DEBUG] - 7z exit code: 0
2017-08-15 19:02:13,221 7720 [INFO ] - C:\Users\buildbot\AppData\Local\Temp\chocolatey\OpenSSHTemp
2017-08-15 19:02:14,188 7720 [ERROR] - ERROR: Command execution stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: ERROR:
2017-08-15 19:02:14,266 7720 [DEBUG] - Built-in PowerShell host called with ['[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\openssh\tools\chocolateyinstall.ps1' -installArguments '' -packageParameters '''] exited with '-1'.

choco version 0.10.7
winver Windows 7 Enterprise (Version 6.1 (Build 7601: Service Pack 1)

**Comment by cowlinator** [comment]
This happens 100% of the time on one computer.

**Comment by DarwinJS** [comment]
It's pretty hard to diagnose this when it's just one machine and the logs aren't showing much.  

Does this buildbot user have administrator permissions and is the process elevated that runs the package?
Is this running under a service or a regular interactive logon?
Does this happen with other chocolatey packages when executed exactly the same way?

Some possible cleanup that might help:
I would suggest that you run "cuninst -y openssh" (even though it says it didn't install).
Then completely delete these folders if any of them exist:
"c:\Program Files\OpenSSH-WIn64"
"c:\ProgramData\ChocolateyInstall\lib\openssh"
"c:\ProgramData\ChocolateyInstall\lib-bad\openssh"
"c:\ProgramData\ChocolateyInstall\lib-bkup\openssh"
"C:\Users\buildbot\AppData\Local\Temp\chocolatey\OpenSSHTemp"

**Comment by cowlinator** [comment]
I'm sorry for the slow response.

The user does have administrator permission, and the command prompt that runs the command is elevated.  I'm running it on a regular interactive logon.

I tried uninstalling through chocolatey, and deleting "C:\Users\buildbot\AppData\Local\Temp\chocolatey\OpenSSHTemp" (none of the others exist).

The problem persists.

I just installed "javaruntime" and "notepadplusplus.install" the same way, and there were no errors.
